utilities/ride-model-util/src/main/java/com/adobe/ride/utilities/model/ModelObject.java

```diff
@@ -278,6 +278,43 @@ private void initializeModelObject(String resourceLocation, JSONObject presetNod
     loadModel(resourceLocation);
   }
 
+  /**
+   * Create a model object from a json schema string and use a predefined set of nodes to start
+   * with. Also only build a targeted set of nodes. The specified resource location is only used as
+   * metadata, so the content will not be fetched from it.
+   * 
+   * @param resourceLocation location of the schema within the project resources.
+   * @param modelString string which conforms to JSON schema standards
+   * @param presetNodes set of JSON nodes and values to be used in the object, not generated
+   *        dynamically
+   * @param nodesToBuild set of JSONnodes which are to be dynamically generated. Remaining schema
+   *        defined nodes will not be generated.
+   * @param useRequiredOnly boolean to determine whether to build only the required json nodes as
+   *        defined in the spec
+   */
+  public ModelObject(String resourceLocation, String modelString, JSONObject presetNodes,
```

Comment:
Don't need resourceLocation here.  That is reference to a json schema file on disk.  My want to take a look at this constructor as well: https://github.com/adobe/ride/blob/develop/utilities/ride-model-util/src/main/java/com/adobe/ride/utilities/model/ModelObject.java#L288

Reply:
I did it for there is already a constructor with the (String, JsonObject, Set<String>, boolean) signature. I had to find a new overload to avoid conflict, and this was the best compromise I've found that didn't require new types (as in wrapping the modelString in a new class). What do you suggest? Thanks for the reviews by the way.

Reply:
Hmm, I don't see the existing signature that you describe.  Can you link me to it in the code?

Reply:
Here it is: 
https://github.com/adobe/ride/blob/89c40d009b95e94e8d46790004629cae41392299/utilities/ride-model-util/src/main/java/com/adobe/ride/utilities/model/ModelObject.java#L260

Reply:
Ah, yes.  If you remove that property, it does collide.  Drat.

I guess we'll have to go with a static constructor.  Something like "createFromSchemaString".  Eventually, these constructors may need some refactoring, but a static constructor can fill this need in the near term.

Reply:
Sure, will do.

Reply:
Done!